FreeRTOS/FreeRTOS
# Remove deprecated ipconfigRAND32

Description
-----------
The macro's ipconfigRAND32() and configRAND32() are not in use anymore. In TCP V2.2.1 releases the ipconfigRAND32 macro was replaced by xApplicationGetRandomNumber().
Removing the instances of ipconfigRAND32 from the code.

Test Steps
-----------
Jenkins run

Checklist:
----------


- [NA] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------
[PR 782](https://github.com/FreeRTOS/FreeRTOS-Plus-TCP/pull/782)


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
